graphql_api/types/account/account.py

```diff
@@ -29,3 +37,20 @@ def resolve_total_seat_count(account: Account, info: GraphQLResolveInfo) -> int:
 @sync_to_async
 def resolve_activated_user_count(account: Account, info: GraphQLResolveInfo) -> int:
     return account.activated_user_count
+
+
+@account_bindable.field("organizations")
+@convert_kwargs_to_snake_case
+def resolve_organizations(
+    account: Account,
+    info: GraphQLResolveInfo,
+    ordering=AccountOrganizationOrdering.ACTIVATED_USERS,
+    ordering_direction=OrderingDirection.DESC,
+    **kwargs,
+):
+    return queryset_to_connection(
+        account.organizations,
+        ordering=(ordering,),
```

Comment:
do we need the tuple here for ordering? vs ordering=ordering

Reply:
Yes we do. Not sure exactly how this works under the hood, but the queryset_to_connection helper does a `for field in ordering`. Some other calls do pass multiple values in the tuple, so seems like there's a reason for this.

Reply:
Got it 👌 Thanks for clarifying!